docs/knowledge_base/tips/css_style_kit/index.md

```diff
@@ -0,0 +1,76 @@
+
+Prepare for a big change in your Taipy applications! 
+The new CSS Style Kit is here and it will make your application look even better. 
```

Comment:
"new CSS Style Kit" -> Stylekit
That's the wording in the doc, please use it.\
And it's not new.